test/e2e/config/vsphere.yaml

```diff
@@ -271,7 +271,7 @@ variables:
   KUBERNETES_VERSION_UPGRADE_FROM: "v1.29.0"
   KUBERNETES_VERSION_UPGRADE_TO: "v1.30.0"
   KUBERNETES_VERSION_LATEST_CI: "ci/latest-1.31"
-  CPI_IMAGE_K8S_VERSION: "v1.31.0-rc.0"
+  CPI_IMAGE_K8S_VERSION: "v1.31.0-rc.1"
```

Comment:
There is no rc.0 image

/approve cancel
/lgtm cancel

Reply:
Yup waiting for https://github.com/kubernetes/k8s.io/pull/7138

Reply:
cc: @XudongLiuHarold

Reply:
Hi team, the PR just merged, you should be able to pull `v1.31.0-rc.1` now :)